hooks-admin.php

```diff
@@ -356,3 +356,7 @@
 // Theme check
 add_action( 'admin_init', '\Pressbooks\Theme\check_required_themes' );
 add_action( 'admin_init', '\Pressbooks\Theme\check_upgraded_customcss' );
+
+if ( $is_book ) {
```

Comment:
I know this is my fault, but `$is_book` logic should be moved into `hooks()`. Example: 

https://github.com/pressbooks/pressbooks-saml-sso/blob/6c0fa5d1154d7d5cdeafbfd9ad400711c8659413/inc/class-updates.php#L27

This is so the class can decide where to print the menu or not.  Useful if we extend this feature to be in both Book and Network Admin.

Reply:
`$is_book` check moved into `hooks`